client/app/telecom/pack/xdsl/modem/templates/pack-xdsl-modem-templates.controller.js

```diff
@@ -46,11 +46,17 @@ angular.module('managerApp').controller('XdslModemTemplateCtrl', class XdslModem
       this.templateName = null;
       this.getModemTemplates();
     }).catch((err) => {
+      let errorMessage = '';
+      if (err.data.message.includes('TemplateName')) {
```

Comment:
`TemplateName` and `Invalid` could be constants

Reply:
yep, it could be.